### ISSUE TITLE
Keras softmax does not align with tensorflow (python)

### ISSUE THREAD

**Issue by AsakusaRinne** [SciSharp/TensorFlow.NET#985]
The Model is quite simple:

```cs
var model = keras.Sequential(new List<Tensorflow.Keras.ILayer>()
{
    keras.layers.InputLayer(128),
    keras.layers.Dense(64),
    keras.layers.Dense(6),
    //keras.layers.Softmax(-1)
});
```

When using `Softmax`, it does not converge, while it converge well after removing it.

In tensorflow (python), the it's just the opposite. The model converge with softmax but does not without softmax.

Both in csharp and python I used `sparse_categorical_crossentropy` loss and `Adam` optimizer.
